sources/subs/BBC/ParserWrapper.php

```diff
@@ -345,65 +362,75 @@ public function setDisabled(array $disabled)
 	}
 
 	/**
+	 * Return the bbc code definitions for the parser
+	 *
 	 * @return Codes
 	 */
 	public function getCodes()
 	{
 		if ($this->codes === null)
 		{
-			$this->codes = new \BBC\Codes(array(), $this->disabled);
+			$this->codes = new Codes(array(), $this->disabled);
 		}
 
 		return $this->codes;
 	}
 
 	/**
+	 * Return an instance of the bbc parser
+	 *
 	 * @return BBCParser
 	 */
 	public function getBBCParser()
 	{
 		if ($this->bbc_parser === null)
 		{
-			$this->bbc_parser = new \BBC\BBCParser($this->getCodes(), $this->getAutolinkParser());
+			$this->bbc_parser = new BBCParser($this->getCodes(), $this->getAutolinkParser());
 		}
 
 		return $this->bbc_parser;
 	}
 
 	/**
+	 * Return and instance of the autolink parser
```

Comment:
Lmao... just don't blame your father. It's not easy being Mr Potatohead
On Dec 18, 2015 8:27 AM, "Spuds" <notifications@github.com> wrote:

> In sources/subs/BBC/ParserWrapper.php
> <https://github.com/elkarte/Elkarte/pull/2315#discussion_r48024293>:
>
> >  		}
> >
> >  		return $this->bbc_parser;
> >  	}
> >
> >  	/**
> > +	 * Return and instance of the autolink parser
>
> Well first I'm a victim of a poorly funded public educational system.
> Second grammatically incorrect is still better then nothing at all [image:
> :innocent:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elkarte/Elkarte/pull/2315/files#r48024293>.
>